### PR TITLE
Palindromes

### PR DESCRIPTION
**Pull request by itsnichi4**
working?

### CODE REVIEW

File: 09_palindromes/palindromes.js [hunk 1]
```diff
@@ -1,4 +1,20 @@
-const palindromes = function () {
+const palindromes = function (str) {
+
+    let cleanStr = str.replace(/\W/g, '');
+    let upperStr = cleanStr.toUpperCase()
```
Comment (cmg193) [inline 09_palindromes/palindromes.js]: You can also just use the cleanStr, no need to create a new variable:

```
let cleanStr = str.replace(/\W/g, '');
cleanStr = cleanStr.toUpperCase()
```

**Review by cmg193 — APPROVED** [review]

**Review by cmg193 — APPROVED** [review]

File: 09_palindromes/palindromes.js [hunk 2]
```diff
@@ -1,4 +1,20 @@
...
+
+    }
+
+    if (upperStr != reverseStr) {
+        return false
+    }
+    else {
+        return true
+    }
+
```
Comment (cmg193) [inline 09_palindromes/palindromes.js]: Return the if condition itself!
`return upperStr != reverseStr`

**Review by cmg193 — CHANGES_REQUESTED** [review]

File: 09_palindromes/palindromes.js (hunk 1 above)
Comment (itsnichi4) [reply 09_palindromes/palindromes.js]: done

File: 09_palindromes/palindromes.js (hunk 2 above)
Comment (itsnichi4) [reply 09_palindromes/palindromes.js]: done

**Review by cmg193 — APPROVED** [review]

**Pull request merged**